azure-pipelines-template.yml

```diff
@@ -0,0 +1,19 @@
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '8.9'
```

Comment:
Can this be templated so we get a node `8` and a latest stable node build, like the Travis yaml specifies? We don't need to test every version on every platform; but checking both versions in addition to each platform would be nice (I'd check each platform on latest stable and 'nix only for node `8`).

On a related note: are there any fast build options we can opt in to? Like Travis had a container opt in (sudo: false); from what I understand the new non-preview 'nix images aren't containerized builds anymore. Mostly just curious here.

Reply:
@weswigham I updated so that it gives the latest version of 11, unfortunately I don't think the node tool installer can check for the latest major version though (it will automatically bump with minor versions now though). Also, it looks like Windows builds are failing on Node 11, npm install seems to fail, it works fine on node 8 though.

Reply:
https://dev.azure.com/TypeScript-TmLanguage/TypeScript-TmLanguage/_build/results?buildId=28&view=logs

Reply:
Can the vm be provisioned with python 2 instead of python 3?

Reply:
@weswigham when you get a chance would you mind taking a look at this again? I think it should be good to go.